src/components/Submenu.js

```diff
@@ -1,11 +1,24 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/books';
 
-const BookItem = () => (
-  <div>
-    <button type="button">Comment</button>
-    <button type="button">Remove</button>
-    <button type="button">Edit</button>
-  </div>
-);
+const Submenu = (prop) => {
+  const dispatch = useDispatch();
+  const { book } = prop;
 
-export default BookItem;
+  const removeBookFromStore = () => {
+    // e.preventDefault();
+
```

Comment:
- [ ] Same here. I would recommend removing the commented code as it makes the codebase cleaner.